core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorManagerService.js

```diff
@@ -47,7 +47,26 @@ oppia.factory('SimpleEditorManagerService', [
             x: 0
           }
         }
-      }
+      },
+      ItemSelectionInput: {
+        CUSTOMIZATION_ARGS:  {
+          choices: {
+            value: ['<p>Choice 1</p>']
+          },
+          minAllowableSelectionCount: {
+            value: 1
```

Comment:
Let's have this be 2, ditto for max. That way this defaults to something with more than one box.

Reply:
Done!